src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java

```diff
@@ -88,65 +88,41 @@ private void processCompleteFiles(CloudBlobContainer container) {
         );
     }
 
-    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
     private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelope envelope) {
-        boolean deleted = false;
+        boolean deleted;
+
+        String loggingContext = "File name: " + envelope.getZipFileName() + ", Container: " + container.getName();
```

Comment:
`String.format` or like this ^? I don't mind really

Reply:
won't fit into one line ;)